views/partials/subscribe_modal/confirm_your_account.hbs

```diff
@@ -3,5 +3,5 @@
   <img class="confirm-your-account-icon" src="/img/confirm-your-account.svg" />
   <p>Check your email for the verification link sent to <span id="submitted-email" class="medium"></span></p>
   <button id="resend-data" class="resend-data internal-link">Not in inbox or spam folder? Resend</button>
```

Comment:
Does this button need GA analytics, or is it handled elsewhere?

Reply:
👍 to getting these "not in inbox" analytics

Reply:
https://github.com/mozilla/blurts-server/blob/6209f244a6513cfd58756b02bcd2d4dca4dadf7e/public/js/monitor.js#L241

Was thinking it made more sense to handle it here in case we decide to send pings for related client-side errors (#384) as well.